scipy/stats/tests/test_continuous_basic.py

```diff
@@ -76,7 +76,7 @@
                'dgamma', 'dweibull', 'erlang', 'f', 'foldcauchy', 'gamma',
                'gausshyper', 'gengamma', 'genhyperbolic',
                'geninvgauss', 'gennorm', 'genpareto',
-               'halfcauchy', 'halfgennorm', 'invgamma', 'jf_skew_t',
+               'halfcauchy', 'halfgennorm', 'invgamma', 'irwinhall', 'jf_skew_t',
```

Comment:
> Yes, it's not super clear to me why this fails.

If "this" means a complex derivative test, it makes sense to me because BSplines don't support complex inputs.

As for why I would not expect the `fit` method to work, it's because it relies on a generic optimization procedure that does not consider (and can't really handle) the integrality constraint. If `scipy.stats.fit` is failing, I'd need to take a closer look. 

I'd like to get other opinions about the `fit` method being unable to fit the shape parameter. The `fit` method is not guaranteed to work for all possible distributions, data, and guesses, but in this case we are essentially guaranteed that it will fail and will just produce garbage. It would probably be better for users if we raise an error unless the shape parameter is fixed. (It can probably fit location and scale, though.)

Reply:
I've added it to the skip/xfail lists for now